webapp/Dockerfile

```diff
@@ -5,12 +5,12 @@ WORKDIR /app
 #Install the dependencies
 RUN npm install
 
-ARG API_URI="http://20.21.124.62:5000/api"
+ARG API_URI="http://localhost:5000/api"
```

Comment:
If you write localhost here and it works, then I don't know why it does. Apart from that this variable wouldn't be necessary as it is already hardcoded in the code

Reply:
It was originally with localhost in the model repository. I am not entirely sure why but I left it to imitate what was there at the beginning (and that works).